tests/fixtures/Requests/response-parse-test.yml

```diff
@@ -0,0 +1,235 @@
+                          <linguisticSections>
+                              <linguisticSection>
+                                  <language code="FR"/>
+                              </linguisticSection>
+                              <linguisticSection>
+                                  <language code="EN"/>
+                              </linguisticSection>
+                          </linguisticSections>
+                      </originalDocument>
+                      <productRequests>
+                          <productRequest requestedDeadline="2019-11-15T12:00:00+01:00" formatCode="NA" statusCode="ENV" trackChanges="false">
+                              <language code="FR"/>
+                          </productRequest>
+                          <productRequest requestedDeadline="2019-11-15T12:00:00+01:00" formatCode="NA" statusCode="ENV" trackChanges="false">
+                              <language code="EN"/>
+                          </productRequest>
+                          <productRequest formatCode="NA" statusCode="ENV" trackChanges="false">
+                              <language code="BG"/>
+                          </productRequest>
+                          <productRequest requestedDeadline="2019-04-15T12:00:00+02:00" formatCode="NA" statusCode="ENV" trackChanges="false">
+                              <language code="CS"/>
+                          </productRequest>
+                          <productRequest requestedDeadline="2019-11-15T12:00:00+01:00" formatCode="NA" statusCode="ENV" trackChanges="false">
+                              <language code="DE"/>
+                          </productRequest>
+                      </productRequests>
+                      <auxiliaryDocuments>
+                          <auxiliaryDocument format="TXT" type="REF" name="test1.txt" language="ML"/>
+                          <auxiliaryDocument format="TXT" type="REF" name="test1.txt" language="RO"/>
+                      </auxiliaryDocuments>
+                  </return>
+              </ns0:createRequestsResponse>
+          </S:Body>
+      </S:Envelope>
+    request:
+      linguisticRequest:
+        - contacts:
+            contact:
+              - UserId: '1'
+                RoleCode: AUTHOR
+              - UserId: '2'
+                RoleCode: EDITOR
+          generalInfo:
+            Title: Test
+            InternalReference: '1'
+            InternalTechnicalId: '1'
+            RequestedDeadline:
+              time: '2020-02-02'
+              timezone: null
+            Sensitive: false
+            DocumentToBeAdopted: true
+            DecideReference: decideReference
+            SentViaRUE: true
+            DestinationCode: PUBLIC
+            SlaAnnex: ''
+            SlaCommitment: ''
+            Comment: ''
+            OnBehalfOf: ''
+            AccessibleTo: ''
+  expectations:
+    request: |
```

Comment:
This is not being used anywhere...

Reply:
We should removed it then, it might be a leftover.

Reply:
Fixed.

Reply:
Fixed.